.github/workflows/env-vars-configure/action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   REF:
     required: true
     description: 'The ref to get the version from.'
+  POSTFIX:
+    required: false
+    default: "-rc"
```

Comment:
I'm not sure this is a good default value. Why not be explicit when calling this function?

Reply:
Removed, I've converted the `POSTFIX` into an env var

Reply:
I've removed the default value and added the postfix variable to the `create-rc-tag` action.